lib/widget/card_container.dart

```diff
@@ -44,18 +45,40 @@ class CardContainer extends StatelessWidget {
     switch (cards.length) {
       case 2:
         return Row(
-          children: expandedCards,
+          children: <Widget>[
+            SizedBox(
+              width: 16,
```

Comment:
lets try to get away from using hard coded sizes as soon as we can, but its ok for now

Reply:
I know @maks, but I was having some problems rendering properly. So, for now, let's use SizedBox a little bit.